src/ring_theory/algebra.lean

```diff
@@ -101,6 +92,19 @@ by rw [smul_def, smul_def, left_comm]
   (r • x) * y = r • (x * y) :=
 by rw [smul_def, smul_def, mul_assoc]
 
+instance to_module : module R A :=
```

Comment:
```suggestion
instance : module R A :=
```

Reply:
Before the refactor there were two module instances with different names. I picked one.

Reply:
Aah, ok. Well, Travis found some instances that haven't been fixed yet.